bundles/framework/myplaces3/view/PlaceForm.js

```diff
@@ -73,6 +75,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.view.PlaceForm',
                 showLabels: true,
                 disabledButtons: false,
                 onFinish: (values) => {
+                    this.cancelCallback(true, true);
```

Comment:
This is misleading. `this._setNewValues()` calls the savecallback so include the stopDrawing() stuff in the saveCallback.